File: internal/parser/testdata/sample.go
package sample

// Greeter says hello.
type Greeter struct {
	Name string
}

// Greet returns a greeting for the Greeter's Name.
func (g Greeter) Greet() string {
	return "hello, " + g.Name
}

func add(a, b int) int {
	return a + b
}

File: internal/parser/testdata/sample.ts
interface Greeting {
  name: string;
}

class Greeter implements Greeting {
  name: string;

  constructor(name: string) {
    this.name = name;
  }

  greet(): string {
    return `hello, ${this.name}`;
  }
}

function add(a: number, b: number): number {
  return a + b;
}
